js/deer-record.js

```diff
@@ -66,7 +66,7 @@ export default class DeerReport {
         this.id = elem.getAttribute(DEER.ID)
         this.elem = elem
         this.evidence = elem.getAttribute(DEER.EVIDENCE) // inherited to inputs
-        this.context = elem.getAttribute(DEER.CONTEXT) // inherited to inputs
+        this.context = UTILS.processContextSyntax(elem.getAttribute(DEER.CONTEXT)) // inherited to inputs
```

Comment:
Since this part is on the constructor, it may be better to leave it as the raw input from the attribute until added to the record.

Reply:
Yep, we can use that UTILs function later when we set that, wasn't sure which one was best.  This was the one-line-change solution.  Also, `processContextSyntax` could probably be a helper in `DeerReport` like `processRecord()` is.